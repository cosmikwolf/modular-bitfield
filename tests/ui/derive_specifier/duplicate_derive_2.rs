use modular_bitfield::prelude::*;

#[bitfield]
#[derive(Specifier)] #[derive(Specifier)]
pub struct SignedInt {
    sign: bool,
    value: B31,
}

fn main() {}
