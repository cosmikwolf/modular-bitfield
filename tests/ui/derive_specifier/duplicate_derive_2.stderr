error: encountered duplicate `#[derive(Specifier)]` parameter
 --> tests/ui/derive_specifier/duplicate_derive_2.rs:4:31
  |
4 | #[derive(Specifier)] #[derive(Specifier)]
  |                               ^^^^^^^^^

error: previous `#[derive(Specifier)]` parameter here
 --> tests/ui/derive_specifier/duplicate_derive_2.rs:4:10
  |
4 | #[derive(Specifier)] #[derive(Specifier)]
  |          ^^^^^^^^^
